### ISSUE TITLE
Lattice outer feature for OpenMC compatibility

### ISSUE THREAD

**Issue by wbinventor** [mit-crpg/OpenMOC#330]
The OpenMC code includes an `outer` attribute for the `Lattice` class ([link](http://openmc.readthedocs.io/en/stable/usersguide/geometry.html#rectangular-lattices)). At the time of this writing, is the only geometric modeling option in OpenMC which is not supported in OpenMOC. There is no reason the OpenMOC codebase cannot support an `outer` attribute on the `Lattice` class, and it should be readily implementable if so desired. However I no longer have the time to do so, but want to document this minor shortcoming here for posterity.

**Comment by ternence6** [comment]
agree.OpenMOC also can't deal with the  Hexagonal Lattices from geometry made by OpenMC.
